src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -175,6 +173,20 @@ def save_chunks_to_file(
     return file_size
 
 
+def copy_files_to_the_root(folder, target_dir, recursive):
+    list_folder = os.listdir(folder)
+    for file in list_folder:
+        if os.path.isdir(file):
+            if recursive is True:
+                copy_files_to_the_root(file, target_dir, recursive)
+            else:
```

Comment:
I feel tempted to just use the logger. So in case someone wants to copy only the files on the directory root, he wont be stuck with the warning all the time.